.gitignore

```diff
@@ -10,5 +10,5 @@ yarn-error.log*
 *.log
 .history
 .sass-cache
-application.css
-aplication.css.map
\ No newline at end of file
+*.css
+.maquetado/application.css.map
```

Comment:
Remove the first `.` in this line and remove the `application.css.map` file.

Reply:
Yes, try to make git ignore every .map file